[2024-03-06T15:01:43.411-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_crypto_dag.DOGE scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-03-06T15:01:43.422-0300] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_crypto_dag.DOGE scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-03-06T15:01:43.422-0300] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-06T15:01:43.436-0300] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): DOGE> on 2024-01-03 00:00:00+00:00
[2024-03-06T15:01:43.441-0300] {standard_task_runner.py:60} INFO - Started process 92126 to run task
[2024-03-06T15:01:43.443-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'get_crypto_dag', 'DOGE', 'scheduled__2024-01-03T00:00:00+00:00', '--job-id', '335', '--raw', '--subdir', 'DAGS_FOLDER/get_crypto.py', '--cfg-path', '/tmp/tmpwaa022tt']
[2024-03-06T15:01:43.445-0300] {standard_task_runner.py:88} INFO - Job 335: Subtask DOGE
[2024-03-06T15:01:43.512-0300] {task_command.py:423} INFO - Running <TaskInstance: get_crypto_dag.DOGE scheduled__2024-01-03T00:00:00+00:00 [running]> on host BITL0401.
[2024-03-06T15:01:43.606-0300] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='get_crypto_dag' AIRFLOW_CTX_TASK_ID='DOGE' AIRFLOW_CTX_EXECUTION_DATE='2024-01-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-03T00:00:00+00:00'
[2024-03-06T15:01:46.608-0300] {history.py:90} ERROR - DOGE: No timezone found, symbol may be delisted
[2024-03-06T15:01:56.623-0300] {python.py:202} INFO - Done. Returned value was: None
[2024-03-06T15:01:56.640-0300] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=get_crypto_dag, task_id=DOGE, execution_date=20240103T000000, start_date=20240306T180143, end_date=20240306T180156
[2024-03-06T15:01:56.671-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T15:01:56.690-0300] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
